@MonuChaudhary14 , please run `.ci/validation.sh no-error-checkstyle-openrewrite-recipes` on local.

`[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.6.0:check (check) on project checkstyle-openrewrite-recipes: Failed during checkstyle execution: Unable to find configuration file at location: https://raw.githubusercontent.com/checkstyle/checkstyle/checkstyle-13.10.0-SNAPSHOT/config/checkstyle-checks.xml: Could not find resource 'https://raw.githubusercontent.com/checkstyle/checkstyle/checkstyle-13.10.0-SNAPSHOT/config/checkstyle-checks.xml'. -> [Help 1] `

looks like we need to override path config in CLI.

--- a/.ci/validation.sh
+++ b/.ci/validation.sh
@@ -1022,6 +1022,21 @@ no-error-sevntu-checks)
   removeFolderWithProtectedFiles sevntu.checkstyle
   ;;
 
+no-error-checkstyle-openrewrite-recipes)
+  set -e
+  CS_POM_VERSION="$(getCheckstylePomVersion)"
+  echo CS_version: "${CS_POM_VERSION}"
+  ./mvnw -e --no-transfer-progress clean install -Pno-validations
+  echo "Checkout target sources ..."
+  checkout_from https://github.com/checkstyle/checkstyle-openrewrite-recipes.git
+  cd .ci-temp/checkstyle-openrewrite-recipes
+  ./mvnw -e --no-transfer-progress verify -DskipTests -Drewrite.skip=true \
+    -Dcheckstyle.version="${CS_POM_VERSION}" \
+    -Dcheckstyle.config=../../config/checkstyle-checks.xml
+  cd ../
+  removeFolderWithProtectedFiles checkstyle-openrewrite-recipes
+  ;;
+
 no-error-contribution)
   set -e
   CS_POM_VERSION="$(getCheckstylePomVersion)"

--- a/.semaphore/semaphore.yml
+++ b/.semaphore/semaphore.yml
@@ -75,6 +75,7 @@ blocks:
                 - .ci/validation.sh no-error-orekit
                 - .ci/validation.sh no-error-checkstyles-sevntu
                 - .ci/validation.sh no-error-sevntu-checks
+                - .ci/validation.sh no-error-checkstyle-openrewrite-recipes
                 - .ci/validation.sh no-error-contribution
                 - .ci/validation.sh no-error-methods-distance
                 - .ci/validation.sh no-error-equalsverifier

